### DISCUSSION TITLE
The user site can be visited even when the repo has been deleted

### DISCUSSION THREAD

**Discussion by siwwwwu** [community/community#23343] [Other Feature Feedback, Questions, & Ideas]
<p>I have created a user site <em><a href="http://username.github.io" rel="noopener nofollow ugc">username.github.io</a></em> linking to a repo. However, I accidentally deleted the repo before I unpublish it. I thought the <em><a href="http://username.github.io" rel="noopener nofollow ugc">username.github.io</a></em> site will be automatically disabled but somehow it does not. I can still visit the page but cannot modify it because there is no repo now!</p>
<p>Does anyone have an idea about how to solve this? Thank you very much!</p>

**Comment by yoannchaudet** [comment]
<p>Hello and thank you for posting here,</p>
<p>Pages sites are automatically deleted upon repository deletion. This is automatic and you don’t need to do anything.</p>
<p>Looking at the repositories you deleted under your account, I understand where your confusion is coming from.</p>
<p>You named your repositories: <code>xiwu.github.io</code> and <code>wuxi.github.io</code> but your GitHub username is <code>siwwwwu</code>. Your user Pages site can only be created in a repository named <code>siwwwwu.github.io</code>. Both <code>xiwu</code> and <code>wuxi</code> are other GitHub accounts that may (or may not) have an existing user Pages site you have no control over.</p>
<p>I hope this clarifies the situation!</p>

**Comment by siwwwwu** [comment]
<p>I see what the problem is. Thank you very much!</p>
